test/test_browser.py

```diff
@@ -5549,6 +5554,69 @@ def test_webpack(self):
     shutil.copyfile('webpack/src/hello.wasm', 'webpack/dist/hello.wasm')
     self.run_browser('webpack/dist/index.html', '/report_result?exit:0')
 
+  def test_fetch_polyfill_shared_lib(self):
```

Comment:
I wonder if we even need to involve shared libraries here since the polyfill is needed even to load the base wasm file (I think).

Reply:
I think its good to have this test either way, that stuff could be changed in the future as the code becomes more unified. We should make sure the code works in all cases.